cql3/CqlLexer.g4

```diff
@@ -0,0 +1,264 @@
+
+lexer grammar CqlLexer;
+
+ 
+LR_BRACKET: '(';
+RR_BRACKET: ')';
+
+LC_BRACKET: '{';
+RC_BRACKET: '}';
+
+LS_BRACKET: '[';
+RS_BRACKET: ']';
+
+
+COMMA: ',';
+SEMI: ';';
+COLON: ':';
+
+SPACE: [ \t\r\n]+ -> channel(HIDDEN);
+SPEC_MYSQL_COMMENT: '/*!' .+? '*/' -> channel(HIDDEN);
+COMMENT_INPUT: '/*' .*? '*/' -> channel(HIDDEN);
+LINE_COMMENT: (
+		('-- ' | '#' | '//') ~[\r\n]* ('\r'? '\n' | EOF)
+		| '--' ('\r'? '\n' | EOF)
+	) -> channel(HIDDEN);
+
+ 
+DOT: '.';
+STAR: '*';
+DIVIDE: '/';
+MODULE: '%';
+PLUS: '+';
+MINUSMINUS: '--';
+MINUS: '-';
+
+
+DQUOTE: '"';
+SQUOTE: '\'';
+ 
+// Keywords
+K_ADD: A D D | 'ADD';
```

Comment:
@KvanTTT ,

Reason for this kind of declarations is that when using grammar to create autocomplete features you need string literal for suggestions.

`K_ADD: A D D; ` matches `add |Add|ADD|adD...` but it can't generate default='ADD' for autocomplete purposes. By adding ` | 'ADD' ` token gets 'real' possible value when traversing through [transitions](https://www.antlr.org/api/Java/org/antlr/v4/runtime/atn/Transition.html). Without it autocomplete won't work. I've declared Keywords that way.

So, parsing wise, `K_ADD: A D D;` is enough. But to use autocomplete (next possible token through transition traversal)  default value has to be added.

Reply:
There are possible ways to fix such workaround:

1. Use [CaseChangingCharStream.java](https://github.com/parrt/antlr4/blob/case-insensitivity-doc/doc/resources/CaseChangingCharStream.java). See detail here: https://github.com/antlr/antlr4/blob/master/doc/case-insensitive-lexing.md
2. Fix the autocompletion algorithm for handling fragment rules. It's harder but more general solution.

Reply:
@KvanTTT ,

You can't fix autocompletition algorithm (2.)  there is nothing wrong with it (in this context).
If you don't have string literal ('SOME_STRING') you can't offer user list of potential tokens that can follow preceding pattern.

Can you imagine that ANTLR4 through [ATNState](https://www.antlr.org/api/Java/org/antlr/v4/runtime/atn/ATNState.html).[getTransitions()](https://www.antlr.org/api/Java/org/antlr/v4/runtime/atn/ATNState.html#getTransitions--) gives you list of all possibilities for pattern `A D D = [aA][bB][cC] ` or   C L U S T E R I N G = [cC][lL][uU][sS][tT][eE][rR][iI][nN][gG] ?

A D D has 8 permutations
C L U S T E R I N G has 1024 permutations
C O L U M N F A M I L Y has 4096 permutations

Hence the fallback string literal. In case of CQL (and SQLs) keywords are uppercase (reason string literal is uppercase) contrary to (most) languages where keywords are lowercased.

There is not need for fix and/or for workarounds.
`K_ADD: A D D | 'ADD';` is legit Antlr 4 grammar syntax. 
This makes this grammar (more) feature complete for potential user.
It is most rational solution  (readable and understandable) at price of  400 bytes overhead in grammar.

Why introduce another lines of code (bugs, maintenance, etc) for no reason.